www/src/pages/index.astro

```diff
@@ -65,42 +65,42 @@ const title = SITE_TITLE;
     <ul class="logos-list">
       {
         [
-          { name: 'Abstract', link: 'https://abstract.com' },
+          { name: 'Abstract', link: 'https://www.goabstract.com' },
           { name: 'Adobe', link: 'https://adobe.com' },
           {
             name: 'Project Wallace',
-            link: 'https://www.github.com/projectwallace/css-design-tokens/',
+            link: 'https://github.com/projectwallace/css-design-tokens/',
           },
           {
             name: 'Design Token Validator (Anima)',
-            link: 'https://www.animaapp.github.io/design-token-validator-site/',
```

Comment:
Including the www subdomain actually breaks the link, GitHub pages returns a 404